app/assets/stylesheets/common/components/more-topics.scss

```diff
@@ -53,6 +53,10 @@
         }
       }
     }
+
+    &-title {
+      display: inline-block;
+    }
```

Comment:
Makes "Related Topics" take up only its text length. Not necessary for this fix but could fix future issues.